src/main/java/Jkbot/command/AddDeadlineCommand.java

```diff
@@ -0,0 +1,37 @@
+package Jkbot.command;
+
+import Jkbot.utils.Storage;
+import Jkbot.utils.TaskList;
+import Jkbot.utils.Ui;
+import Jkbot.exception.JkBotException;
+
+/**
+ * Command to add a new deadline task.
+ */
+public class AddDeadlineCommand implements Command {
```

Comment:
Perhaps a noun class name would be more suitable?
```suggestion
public class DeadlineCommand implements Command {
```

Reply:
I believe the name AddDeadlineCommand is meant to be a noun as it refers to the command to "add a deadline", rather than the action of adding a "deadline command".

However, changing the name to DeadlineCommand would be less confusing and you should consider such a change.